docs-ref-services/servicebus.md

```diff
@@ -2,47 +2,200 @@
 title: Service Bus libraries for Python 
 description: Reference documentation for the Python client and management libraries for Service Bus
 keywords: Azure, Python, SDK, API, messaging, pubsub, pub-sub, message broker
-author: lisawong19
-ms.author: liwong
-manager: routlaw
-ms.date: 02/21/2018
+author: annatisch
+ms.author: antisch
+manager: mayurid
+ms.date: 01/10/2019
 ms.topic: article
 ms.devlang: python
 ms.service: service-bus
 ---
 
 # Service Bus libraries for Python
 
-## Overview
+Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging.
 
-Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging. 
+## What's new in v0.50.0?
+As of version 0.50.0 a new AMQP-based API is available for sending and receiving messages. This update involves **breaking changes**.
+Please read [Migration from v0.21.1 to v0.50.0](#migration-from-v0.21.1-to-v0.50.0) to determine if upgrading is
+right for you at this time.
 
-## Install the libraries
+The new AMQP-based API offers improved message passing reliability, performance and expanded feature support going forward.
+
+For documentation on the legacy HTTP-based operations please see [Using HTTP-based operations of the legacy API](#using-http-based-operations-of-the-legacy-api)
+
```

Comment:
To help discoverability and SEO, add a link to the source code for the SDK at the end of the introduction section.

See [README template](https://github.com/binderjoe/cosmos-python-prototype/blob/master/doc/doc_readme_template.md) and [example README](https://github.com/binderjoe/cosmos-python-prototype/blob/master/README.md).

(I'm going to be staging said guidance in contributors guide today.)